CheckIn/PagerEntry.ascx.cs

```diff
@@ -0,0 +1,225 @@
+                    if ( currentFamily != null )
+                    {
+                        currentFamily.Group.LoadAttributes();
+                    }
+                    else
+                    {
+                        GoBack();
+                    }
+                    var checkinTypes = GetAttributeValues( AttributeKey.CheckinType ).AsGuidList();
+                    if ( checkinTypes.Any() && LocalDeviceConfig.CurrentCheckinTypeId.HasValue )
+                    {
+                        var checkinTypeGroup = GroupTypeCache.Get( LocalDeviceConfig.CurrentCheckinTypeId.Value );
+                        if ( !checkinTypes.Contains( checkinTypeGroup.Guid ) )
+                        {
+                            GoToNextPage();
+                        }
+                    }
+
+
+                    lTitle.Text = GetTitleText();
+                    lbSubmit.Text = "Check In";
+                    lbSubmit.Attributes.Add( "data-loading-text", "Printing..." );
+
+                    tbPagerNumber.Label = GetAttributeValue( AttributeKey.Caption );
+
+                    pnlKeypad.Visible = GetAttributeValue( AttributeKey.DisplayKeypad ).AsBoolean();
+                }
+            }
+        }
+
+        private string GetTitleText()
+        {
+            // The checkinPerson, selectedGroup, and selectedLocation are only needed for individual checkins, so no use running the queries if this is a mutli person checkin.
+            var checkinPerson = CurrentCheckInState.CheckInType.TypeOfCheckin == TypeOfCheckin.Individual
+                ? CurrentCheckInState.CheckIn.Families
+                    .Where( f => f.Selected )
+                    .SelectMany( f => f.People.Where( p => p.Selected ) )
+                    .FirstOrDefault()
+                : null;
+
+            var selectedGroup = checkinPerson?.GroupTypes
+                .Where( t => t.Selected )
+                .SelectMany( t => t.Groups.Where( g => g.Selected ) )
+                .FirstOrDefault();
+
+            var selectedLocation = selectedGroup?.Locations.Where( l => l.Selected ).FirstOrDefault()?.Location;
+
+            var selectedIndividuals = CurrentCheckInState.CheckIn.CurrentFamily.People.Where( p => p.Selected == true ).Select( p => p.Person );
+
+            var mergeFields = new Dictionary<string, object>
+            {
+                { LavaMergeFieldName.Family, CurrentCheckInState.CheckIn.CurrentFamily.Group },
+                { LavaMergeFieldName.SelectedIndividuals, selectedIndividuals },
+                { LavaMergeFieldName.CheckinType, CurrentCheckInState.CheckInType.TypeOfCheckin },
+                { LavaMergeFieldName.SelectedGroup, selectedGroup?.Group },
+                { LavaMergeFieldName.SelectedLocation, selectedLocation },
+            };
+
+            var timeSelectHeaderLavaTemplate = CurrentCheckInState.CheckInType.TimeSelectHeaderLavaTemplate ?? string.Empty;
```

Comment:
Pull actual block template attribute here instead of timeselectheaderlavatemplate.